frontend/src/stimulus/controllers/disable-when-checked.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { ApplicationController } from 'stimulus-use';
+
+export default class OpDisableWhenCheckedController extends ApplicationController {
+  static targets = ['cause', 'effect'];
+
+  static values = {
+    reversed: Boolean,
+  };
+
+  declare reversedValue:boolean;
+
+  declare readonly hasReversedValue:boolean;
+
+  declare readonly effectTargets:HTMLInputElement[];
+
+  causeTargetConnected(target:HTMLElement) {
+    target.addEventListener('change', this.handleChange.bind(this));
+  }
+
+  causeTargetDisconnected(target:HTMLElement) {
+    target.removeEventListener('change', this.handleChange.bind(this));
+  }
+
+  private handleChange(evt:InputEvent):void {
```

Comment:
Matter of preference I guess but describing what is done in the method is preferable to me (e.g. `toggleDisabled` or `disableWhenChecked`)